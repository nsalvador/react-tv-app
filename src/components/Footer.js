import React from 'react';

const Footer = () => (
	<footer>
		<p>Created by Nigel D. Salvador, &copy; {new Date().getFullYear()}</p>
	</footer>
);

export default Footer;
