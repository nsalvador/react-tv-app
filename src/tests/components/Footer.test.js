import React from 'react';
import { shallow } from 'enzyme';
// import ReactShallowRenderer from 'react-test-renderer/shallow';
import Footer from '../../components/Footer';

it('should render Footer correctly', () => {
	const wrapper = shallow(<Footer />);
	expect(wrapper).toMatchSnapshot();
	// const renderer = new ReactShallowRenderer();
	// renderer.render(<Footer />);
	// expect(renderer.getRenderOutput()).toMatchSnapshot();
});
