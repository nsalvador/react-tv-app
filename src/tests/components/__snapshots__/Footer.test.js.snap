// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Footer correctly 1`] = `
<footer>
  <p>
    Created by Nigel D. Salvador, Â© 
    2020
  </p>
</footer>
`;
